src/libostree/ostree-repo-refs.c

```diff
@@ -779,6 +779,126 @@ ostree_repo_remote_list_refs (OstreeRepo       *self,
   return TRUE;
 }
 
+#ifdef OSTREE_ENABLE_EXPERIMENTAL_API
+static gboolean
+remote_list_collection_refs_process_refs (OstreeRepo   *self,
+                                          const gchar  *remote_name,
+                                          const gchar  *summary_collection_id,
+                                          GVariant     *summary_refs,
+                                          GHashTable   *ret_all_refs,
+                                          GError      **error)
+{
+  gsize j, n;
```

Comment:
I'm OK with this as is, but a lot of the libostree codebase continues to move to decl+init inline, so it'll probably be converted at some point.